qualification/report/generate_pdf.py

```diff
@@ -243,7 +243,8 @@ class CBFConfiguration:
 
     mode_config: dict
     uuid: UUID
-    tasks: dict[str, Task]
+    tasks: dict[str, Task]  # Empty if the CBF failed
+    error: str | None  # Empty if the CBF started successfully
```

Comment:
Hate to be pedantic, but will it 'Empty' or `None`? i.e. the actual value.
* I see on Line 564 you don't have a default for the `msg.get("error")`, 
* So I'm leaning towards `None` if the CBF started successfully

Reply:
Good catch - empty is not the same as None. I'll fix.